### PR TITLE
Trevor Fehrman

### PR DESCRIPTION
**Pull request by trevorfehrman**

### CODE REVIEW

File: basic-javascript/project-1.spec.js [hunk 1]
```diff
@@ -1,3 +1,157 @@
...
+	);
+});
+
+it.skip("Should get the area of a circle", () => {
+	expect(helpers.getCircleArea(10)).toBe(Math.PI * 100);
+	expect(helpers.getCircleArea([10])).toBe("Passed argument isn't a number");
+});
+
+it("Should get the volume of a rectangular prism", () => {
+	expect(helpers.getRectangularPrismVolume(10, 10, 10)).toBe(1000);
```
Comment (kelfro) [inline basic-javascript/project-1.spec.js]: All of your tests are marked to skip except this one.

**Comment by kelfro** [comment]
Great job completing MVP! I like how you edited the main project files to help sanitize the data. Maybe today you can try to work on some more edge cases before moving on to stretch.

**Comment by kelfro** [comment]
Love your work on the game project! Nice job setting it up. All of your tests look well-written and they all pass, including the async. Well done :)
